### PR TITLE
CI Update - .NET 7 RC 1

### PR DESCRIPTION
**Pull request by gewarren**
Build.Reason:Manual by Genevieve Warren
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=318563&view=results
 source_repo.branch:net7-rc1
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

### CODE REVIEW

File: xml/Microsoft.Extensions.DependencyInjection/DefaultServiceProviderFactory.xml [hunk 1]
```diff
@@ -25,6 +25,12 @@
...
   </Interfaces>
+  <Attributes>
+    <Attribute FrameworkAlternate="dotnet-plat-ext-7.0">
+      <AttributeName Language="C#">[System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Using Microsoft.Extensions.DependencyInjection requires generating code dynamically at runtime. For example, when using enumerable and generic ValueType services.")]</AttributeName>
+      <AttributeName Language="F#">[&lt;System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Using Microsoft.Extensions.DependencyInjection requires generating code dynamically at runtime. For example, when using enumerable and generic ValueType services.")&gt;]</AttributeName>
+    </Attribute>
+  </Attributes>
   <Docs>
     <summary>Default implementation of <see cref="T:Microsoft.Extensions.DependencyInjection.IServiceProviderFactory`1" />.</summary>
     <remarks>To be added.</remarks>
```
Comment (carlossanlop) [inline xml/Microsoft.Extensions.DependencyInjection/DefaultServiceProviderFactory.xml:36]: Line 165 of this file. Could the suggestion be fixed by forcing the inheritdoc to point to the specific cref of the interface this API implements?:

```
        <inheritdoc cref="M:Microsoft.Extensions.DependencyInjection.IServiceProviderFactory`1.CreateServiceProvider(`0)" />
```

**Comment by carlossanlop** [comment]
> File xml/System.Text.RegularExpressions/RegexGeneratorAttribute.xml was deleted without redirection. To avoid broken links, add a redirection.

Where does this error come from? Is this deleted file being referenced somewhere?

**Comment by gewarren** [comment]
> > File xml/System.Text.RegularExpressions/RegexGeneratorAttribute.xml was deleted without redirection. To avoid broken links, add a redirection.
> 
> Where does this error come from? Is this deleted file being referenced somewhere?

You can ignore it. It's because that type was renamed to GeneratedRegexAttribute.

File: xml/Microsoft.Extensions.DependencyInjection/DefaultServiceProviderFactory.xml (hunk 1 above)
Comment (gewarren) [reply xml/Microsoft.Extensions.DependencyInjection/DefaultServiceProviderFactory.xml:36]: Worth a try - or we could just delete the `<inheritdoc />` line.

**Comment by carlossanlop** [comment]
Tar changes included. Merge conflicts solved. I will generate RC2 intellisense from this, and will generate the undoc API report as well.

**Comment by carlossanlop** [comment]
Incoming PRs: https://github.com/dotnet/dotnet-api-docs/pull/8390 and https://github.com/dotnet/dotnet-api-docs/pull/8391

File: xml/System.Windows.Forms/Control.xml [hunk 2]
```diff
@@ -3602,6 +3604,67 @@ The following table lists Windows Forms controls and which event (<xref:System.W
...
+        <Attribute FrameworkAlternate="windowsdesktop-7.0">
+          <AttributeName Language="C#">[System.ComponentModel.Browsable(false)]</AttributeName>
+          <AttributeName Language="F#">[&lt;System.ComponentModel.Browsable(false)&gt;]</AttributeName>
+        </Attribute>
+      </Attributes>
+      <ReturnValue>
+        <ReturnType>System.Object</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Windows.Forms/Control.xml:3633]: @RussKie Will your team bring in missing doc comments before .NET 7 ships?

**Comment by gewarren** [comment]
@carlossanlop Are we good to merge this? If so can you approve?

File: xml/System.Windows.Forms/Control.xml (hunk 2 above)
Comment (carlossanlop) [reply xml/System.Windows.Forms/Control.xml:3633]: @singhashish-wpf and @pchaurasia14 did you port docs for WPF?

**Review by carlossanlop — APPROVED** [review]
This is good from the perspective of runtime APIs. If WPF/WinForms need to add extra documentation, they can do it directly in main after we merge this.

File: xml/System.Windows.Forms/Control.xml (hunk 2 above)
Comment (gewarren) [reply xml/System.Windows.Forms/Control.xml:3633]: I'm going to resolve this conversation just so I can merge the PR. I believe that Carlos is going to port the docs in a separate PR.

**Comment by carlossanlop** [comment]
Don't merge this yet, please. The WinForms PR is targeting this branch: https://github.com/dotnet/dotnet-api-docs/pull/8393

**Comment by carlossanlop** [comment]
Nevermind, automerge 😄

**Comment by amzwzw** [comment]
Iove Ahmed

**Pull request merged**